index.html

```diff
@@ -0,0 +1,291 @@
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>CC Exhibition</span>
+              <p>
+                Appreciate various creations applying CC license of participants
+                from this community
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/ccIgnite.svg"
+                  alt="ccIgnite"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>Forum</span>
+              <p>
+                Have the time to share your thoughts and opinionswith experts
+                for each topic
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/workshop.svg"
+                  alt="workshop"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>Workshop</span>
+              <p>
+                Denotaing and small community using open sources and license
+                than sitting
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/workshop.svg"
+                  alt="workshop"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>CC Ignite </span>
+              <p>
+                Get opportunity to network the CC affiliates around the world,
+                also after conference
+              </p>
+            </div>
+          </div>
+          <a class="sectiontwo__container__title display" href="#"
+            >SEE THE WHOLE PROGRAM</a
+          >
+          <a class="sectiontwo__container__title" href="#" id="second">
+            Join unicef's annual human support</a
+          >
+        </div>
+      </section>
+      <!-- end sectiontwo -->
+      <!-- start sectionthree -->
+      <section class="sectionthree pd-y" id="sectionthree">
+        <div class="container">
+          <h3 class="heading">Featured Speakers</h3>
+          <span class="line"></span>
+          <div class="sectionthree__container__speakers"></div>
+        </div>
+      </section>
+      <!-- end sectionthree -->
+      <!-- start sectionfour -->
+      <section class="sectionfour" id="sectionfour">
+        <div class="container">
+          <h3 class="heading">Partners</h3>
```

Comment:
- [ ] This text is not visible as it does not make use of the correct text colors. Kindly change it.

**This is how I am seeing it:**

![image](https://user-images.githubusercontent.com/86318284/197013955-833fa0c3-cfd5-44e7-b9f0-392233aba3b7.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/197014030-da550d1f-a4d7-4eaf-ae1d-d79839dc8872.png)